app/screens/settings/about/about.tsx

```diff
@@ -171,6 +184,22 @@ const About = ({componentId, config, license}: AboutProps) => {
         popTopScreen(componentId);
     });
 
+    const copyToClipboard = useCallback(
+        () => {
+            const appVersion = `${intl.formatMessage({id: 'settings.about.version', defaultMessage: 'App Version:'})} ${intl.formatMessage({id: 'settings.about.build', defaultMessage: '{version} (Build {number})'},
```

Comment:
I just realized something here. It is better if you keep everything in one single message. So something like...
`{id: 'settings.about.version', defaultMessage: 'App Version: {version} (Build {number})'}`. That way you can omit the outermost `${}` and hopefully it is a bit more readable. The reason behind this is that in some languages may make sense to somehow reorder these in some way (Right to Left languages come to mind, but not sure if would be really the case here).
Would you mind following the same approach with the 4 variables you are declaring here?